src/App.tsx

```diff
@@ -8,13 +8,43 @@ import {
   MovieDetails,
   SortOption,
   Movie,
+  Dialog,
 } from "./components";
 import { useState } from "react";
 import { genres } from "./components/GenreSelect/Component/GenreSelect";
-
+import MovieForm from "./components/MovieForm/Component/MovieForm";
+import styles from "./App.module.css";
 function App() {
   const [selectedGenre, setSelectedGenre] = useState<Genre>(genres[0]);
   const [selectedSort, setSelectedSort] = useState<SortOption>();
+  const [isOpenAdd, setIsOpenAdd] = useState(false);
+  const [isOpenEdit, setIsOpenEdit] = useState(false);
+  const [isOpenDeleteConfirmation, setIsOpenEditDeleteConfirmation] =
+    useState(false);
+
+  const openDialogAdd = () => {
+    setIsOpenAdd(true);
+  };
+
+  const closeDialogAdd = () => {
+    setIsOpenAdd(false);
+  };
+
+  const openDialogEdit = () => {
```

Comment:
Usually for such handlers 'show...' and 'hide...' naming pattern is used. It is more semantically precise.